event.c

```diff
@@ -396,7 +396,27 @@ event_handle_configurerequest(xcb_configure_request_event_t *ev)
         }
 
         c->got_configure_request = true;
-        client_resize(c, geometry, false);
+
+        /* Request the changes to be applied */
+        luaA_object_push(L, c);
+        lua_pushstring(L, "ewmh");     /* context */
```

Comment:
It doesn't matter much and we already get this wrong in other places, but: This is either core X11 or ICCCM, but not EWMH. But whatever...